docs/openapi3.yaml

```diff
@@ -19,39 +19,43 @@ info:
     For an introduction to Predator and its concepts, see the <a href="https://www.zooz.com" target="_blank">Predator Documentation</a>.
 
 tags:
-  - name: DSL Definitions
-    description: |
-      Predator's Domain Specific Language (DSL) allows you to generate request templates, which you can then reuse in the same test and in other tests, reducing replication.
-  - name: Tests
-    description: |
-      Tests include end-to-end scenarios that are executed at pre-configured intervals to provide in-depth performance metrics of your API.
-  - name: Jobs
-    description: |
-      Predator executes tests through so-called **jobs**. Depending on your configuration, the job will either execute immediately or at scheduled intervals.
-  - name: Reports
-    description: |
-      Reports give you insight into the performance of your API. Predator generates a report for each test that is executed.
-  - name: Configuration
-    description: |
-      This resource allows you to configure Predator programmatically.
+- name: DSL Definitions
+  description: |
+    Predator's Domain Specific Language (DSL) allows you to generate request templates, which you can then reuse in the same test and in other tests, reducing replication.
+- name: Tests
+  description: |
+    Tests include end-to-end scenarios that are executed at pre-configured intervals to provide in-depth performance metrics of your API.
+- name: Processors
+  description: |
+    Processor file allows Preadtor execute custom javascript code.
```

Comment:
Processor files allow Predator to execute custom javascript code during test flows.